__tests__/app.test.js

```diff
@@ -0,0 +1,37 @@
+const request = require("supertest");
+const app = require("../ExpressApp/app");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data/index");
+const db = require("../db/connection");
+
+beforeEach(() => seed(data));
+afterAll(() => db.end());
+
+describe("API", () => {
+  test("200: GET /api/healthcheck response with an okay message", () => {
+    return request(app).get("/api/healthcheck").expect(200);
+  });
+
+  test("404: GET /api/banana should return not found", () => {
+    return request(app)
+      .get("/api/banana")
+      .expect(404)
+      .expect({ msg: "Not Found" });
+  });
+});
+
+describe("/api/topics", () => {
+  test("200: GET /api/topics should respond with all available topics", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        body.topics.forEach((topic) => {
```

Comment:
Great test so far! Bearing in mind that if the array that comes back is empty - this test would still pass. How would you need to update the test to make sure that it only passes when we get some topics back?